openedx/features/course_experience/plugins.py

```diff
@@ -35,11 +36,14 @@ def icon_classes(cls):
     @classmethod
     def is_enabled(cls, request, course_key):
         """
-        Returns True if this tool is enabled for the specified course key.
+        Returns True if the user should be shown course updates for this course.
         """
+        if not UNIFIED_COURSE_TAB_FLAG.is_enabled(course_key):
+            return False
```

Comment:
I forgot about your request for that other test... I'll add one.

I'm not sure why I added this logic. It now makes sense to me that the updates should be shown the same way regardless of the value of the flag, so I'll remove that item.